### PR TITLE
Exposing matchers at the top level

### PR DESCRIPTION
**Pull request by DavidTanner**
tibbs

Exposing the types gets a little tricky, I can add to the Readme once we have everything smoothed out.

### CODE REVIEW

**Comment by jonathanbranam** [comment]
I still don't see a pre-release published. Am I looking in the right place? Versions on this page: https://www.npmjs.com/package/@lifeomic/jest-config

File: test/methods.test.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+test('has exposed some desired methods', () => {
+  expect({
+    numberVal: .31,
+  }).toEqual({
+    numberVal: expect.expect('toBeCloseTo', 0.3, 1),
```
Comment (shawnzhu) [inline test/methods.test.ts:5]: question: do we want:
```Typescript
numberVal: expect.expect('toBeCloseTo', number, number);
```
Or:
```Typescript
numberVal: expect.toBeCloseTo(number, number);
```

Comment (DavidTanner) [reply test/methods.test.ts:5]: I'm not sure how it would be adding each of these to the root level, but I guess we could do it that way.

**Comment by DavidTanner** [comment]
Looks like the github action didn't run...

**Review by IevgenRagulin — APPROVED** [review]

**Comment by jonathanbranam** [comment]
I have:
```package.json
    "@lifeomic/jest-config": "0.0.1-pr-10-1668724868-1641578826.0",
```
and
```jest.config.js
  preset: '@lifeomic/jest-config',
```

But I get a typescript error compiling:

```
tsc --noEmit
test/unit/utils/patientStats.test.ts:284:25 - error TS2339: Property 'toBeCloseTo' does not exist on type 'Expect'.

284         average: expect.toBeCloseTo(51.67),
                            ~~~~~~~~~~~
```

**Comment by jonathanbranam** [comment]
It looks like I got it. I just need to import the jest-config in my test setup somewhere:

```
import '@lifeomic/jest-config';
```

**Review by jonathanbranam — APPROVED** [review]
Works great! Thank you!!

**Comment by DavidTanner** [comment]
@jonathanbranam the recommended way to import it is inside the jest config.

```
setupFilesAfterEnv: ['@lifeomic/jest-config']
```

**Comment by jonathanbranam** [comment]
> @jonathanbranam the recommended way to import it is inside the jest config.
> 
> ```
> setupFilesAfterEnv: ['@lifeomic/jest-config']
> ```

`tsc` doesn't seem to find that, though, when run outside of the context of `jest`, e.g. as a step in `pretest`.

**Pull request merged**